ui/src/components/Filter.tsx

```diff
@@ -56,12 +55,18 @@ const Tagline = styled.span`
   padding-bottom: 16px;
 `;
 
-export function Filter(): JSX.Element {
+export function Filter({state, setState}): JSX.Element {
```

Comment:
Why do we need to pass `state` into the filter?

Can we just expose `onChange`, and move the `onTagsChange` function up to the parent component? Also please type the props?

so something like:

```
export function Filter({ onChange }: FilterProps): JSX.Element {
```